ec2_metadata.py

```diff
@@ -61,6 +61,28 @@ def mac(self):
         return requests.get(METADATA_URL + 'mac').text
 
     @cached_property
+    def network(self):
```

Comment:
I'm gonna merge this as is and move it on over to an object-based structure with lazy loading of the interfaces' properties so it's more in line with the way the top level object works. Thanks for the contribution 👌